raspberryPi/raspberryPi.py

```diff
@@ -0,0 +1,48 @@
+import requests
+from fractions import Fraction
+from picamera import PiCamera
+from time import sleep
+
+static_url = 'http://35.245.175.140/model_form_upload'
+static_camera_identifier = 'openspot_deployed_camera_001'
+static_image_file = static_camera_identifier + '.jpg'
+static_image_path = '/home/pi/CarDetector/raspberryPi'
+
+def upload_image():
+    with open(static_image_path + static_image_file, "rb") as image:
+        s = requests.Session()
+        # fetch the CSRF cookie
+        r1 = s.get(static_url)
+        if(r1.status_code == 200):
+            csrf_token = r1.cookies["csrftoken"]
+
+            # post stored.csv
+            r2 = s.post(
+                static_url,
+                data={"csrfmiddlewaretoken": csrf_token, "description": static_camera_identifier},
+                files={"image": image},
+            )
+
+
+print("Setting up the camera")
+# Light Mode 
+# camera = PiCamera()
+# camera.resolution = (2592, 1944)
+# camera.framerate = 15
```

Comment:
Does this mean the camera wont take a properly exposed picture during the day? if so could we update this so it toggles between day mode and night mode based off the time?

Reply:
will do later